src/helpers/payout.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Helper functions for payout config
+ *
+ * We support payouts (permit2 + gnosis chain) for 2 networks:
+ * - ethereum mainnet
+ * - gnosis chain
+ *
+ * To support a new chain add RPC URL and payment token to the "getPayoutConfigByChainId()" method.
+ * Notice that chain should be compatible with gelato network, i.e.:
+ * 1. Gelato network should support the added chain (https://docs.gelato.network/developer-services/relay/api#oracles)
+ * 2. Gelato network should support the added payment token (https://docs.gelato.network/developer-services/relay/api#oracles-chainid-paymenttokens)
+ */
+
+import { Static } from "@sinclair/typebox";
+import { DEFAULT_RPC_ENDPOINT } from "../configs";
+import { PayoutConfigSchema } from "../types";
+
+// chain ids
+const CHAIN_GNOSIS = 100;
```

Comment:
I just realized that the interesting chains to integrate with for me personally:

Mainnet: 1
Optimism: 10
Gnosis Chain: 100

Reply:
Adding a new chain is basically 6 lines of code:
```js
const CHAIN_OPTIMISM = 10;
const PAYMENT_TOKEN_CHAIN_OPTIMISM = "0x00"; // some token supported by gelato
if (chainId === CHAIN_OPTIMISM) {
    payoutConfig.rpc = "https://rpc.optimism.com";
    payoutConfig.paymentToken = PAYMENT_TOKEN_CHAIN_OPTIMISP;
}
```

Then a partner would need to:
1. Update `chain-id` in the config
2. [Approve permit2 contract to spend tokens](https://dao.ubq.fi/ubiquibot-setup#42f485283d0d463c8a2c8289ecfb8248)